_config.yml

```diff
@@ -19,7 +19,8 @@ keywords:    "coala, review, analysis, static, bears"
 description: "coala provides a common command-line interface for linting and fixing all your code, regardless of the programming languages you use."
 source_link: "https://github.com/coala-analyzer/coala/"
 favicon:     "https://raw.githubusercontent.com/coala-analyzer/coala-artwork/master/artwork/coala_bear.png"
-
+logo:        "img/coala_logo.svg"
```

Comment:
Alright. I'll use the link of artwork repo. i.e. https://raw.githubusercontent.com/coala/artwork/master/artwork/logo/coala_color.svg